IFC4x3/Templates/Object Connectivity/Product Span Positioning/DocTemplateDefinition.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocTemplateDefinition xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Product Span Positioning" UniqueId="1cdcaa9f-85d5-42cf-9aaf-0fd663b14ee8" Type="IfcProduct">
+	<Rules>
+		<DocModelRuleAttribute Name="PositionedRelativeTo">
```

Comment:
does ther eneed to be some sort of Implied ordering in the IfcReferents?

or will we address this using GU and name labels

Reply:
According to the discussion between me, @czapplitec and @SergejMuhic was that this was a GU issue and to use the Name attribute of IfcRelPositions to capture the ordering (start/end) - hence the rule ids for the names.

Reply:
Awseome, i will put it on the GU work/issue list.